test/framework/ippool.go

```diff
@@ -0,0 +1,28 @@
+package framework
+
+import (
+	"fmt"
+
+	"github.com/aws/eks-anywhere/pkg/networkutils"
+)
+
+func PopIPFromEnv(ipPoolEnvVar string) (string, error) {
+	ipPool, err := networkutils.NewIPPoolFromEnv(ipPoolEnvVar)
+	if err != nil {
+		return "", fmt.Errorf("ip pool does not exist for the provided env var %s", ipPoolEnvVar)
```

Comment:
nit: I would propagate the err and keep the error msg here generic to this function. Something like 
```suggestion
		return "", fmt.Errorf("error popping IP from environment: %v", err)
```
so the error would look like
```
error popping IP from environment: T_CLUSTER_IP_POOL environment ip pool does not exist
```